osu.Game/Tests/Visual/EditorClockTestScene.cs

```diff
@@ -19,14 +20,20 @@ public abstract class EditorClockTestScene : OsuManualInputManagerTestScene
         [Cached]
         private readonly OverlayColourProvider overlayColour = new OverlayColourProvider(OverlayColourScheme.Aquamarine);
 
+        [Cached(typeof(EditorBeatmap))]
+        [Cached(typeof(IBeatSnapProvider))]
+        private readonly EditorBeatmap editorBeatmap;
+
         protected readonly BindableBeatDivisor BeatDivisor = new BindableBeatDivisor();
         protected new readonly EditorClock Clock;
 
         protected virtual bool ScrollUsingMouseWheel => true;
 
         protected EditorClockTestScene()
         {
-            Clock = new EditorClock(new Beatmap(), BeatDivisor) { IsCoupled = false };
+            editorBeatmap = new EditorBeatmap(new Beatmap());
```

Comment:
Is causing test failures, because the `new Beatmap()` inside has a null placeholder ruleset that is not available:

![image](https://user-images.githubusercontent.com/20418176/170328148-975f4960-a110-44bf-88ab-6a02dc47c00d.png)

Reply:
There's a reason we've avoided adding this to this class until now I guess...

Will attempt to move it local to usage.